src/classes/TaskListController.cls

```diff
@@ -0,0 +1,33 @@
+public with sharing class TaskListController {
+	@AuraEnabled
+	public static List<Object> getAllTasks() {
+		List<Object> ltask = [SELECT Id, Name, Description__c, Type__c FROM HTask__c];
+		return ltask;
+	}
+
+	@AuraEnabled
+	public static List<Object> getTODOTasks() {
+		List<Object> ltask = [SELECT Id, Name, Description__c, Type__c
```

Comment:
you could re-factor these 3 methods into a single one that receives the type as parameter.